cmd/status/status.go

```diff
@@ -70,6 +72,29 @@ func runStatus() error {
 		ss.NumFlows,
 		(float64(ss.NumFlows)/float64(ss.MaxFlows))*100,
 	)
+	numConnected := ss.GetNumConnectedNodes()
+	numUnavailable := ss.GetNumUnavailableNodes()
+	if numConnected != nil {
+		total := ""
+		if numUnavailable != nil {
+			total = fmt.Sprintf("/%d", numUnavailable.Value+numConnected.Value)
+		}
+		fmt.Printf("Connected nodes: %d%s\n", numConnected.Value, total)
+	}
+	if numUnavailable != nil {
+		if unavailable := ss.GetUnavailableNodes(); unavailable != nil && numUnavailable.Value > 0 {
+			sort.Strings(unavailable) // it's nicer when displaying unavailable nodes list
+			if numUnavailable.Value > uint32(len(unavailable)) {
+				unavailable = append(unavailable, fmt.Sprintf("and %d more...", numUnavailable.Value-uint32(len(unavailable))))
+			}
+			fmt.Printf("Unavailable nodes: %d\n  - %s\n",
```

Comment:
nit: We have "Max/Current Flows" (capital case). Should we have "Unavailable Nodes" here as well?

Reply:
Yup, fixed.